lokeshjain2008/darpo-infra
# Consolidate outputs into a single file

## Description

This PR fixes the duplicate outputs issue by consolidating all outputs into a single `outputs.tf` file and removing the separate `debug_outputs.tf` file.

### Changes:

1. **Consolidated Outputs**:
   - Moved all outputs to `outputs.tf`
   - Organized by service (ECR, EKS, RDS)
   - Kept debug outputs for troubleshooting
   - Added clear descriptions

2. **Removed File**:
   - Deleted `debug_outputs.tf` to avoid duplicates

### Output Organization:

The outputs are now organized into sections:
- ECR outputs (repository URL)
- EKS outputs (cluster endpoint, name)
- RDS outputs (endpoint, database name)
- Debug outputs (VPC IDs, subnet groups)

## Testing Instructions

1. Remove the old debug_outputs.tf:
```bash
rm terraform/environments/dev/debug_outputs.tf
```

2. Apply the changes:
```bash
cd terraform/environments/dev
terraform plan  # Should show no duplicates
terraform apply
```

3. Verify outputs:
```bash
terraform output  # Should show all outputs properly
```
